### DISCUSSION TITLE
Failed to install packages

### DISCUSSION THREAD

**Discussion by BabakAmini** [refinedev/refine#5575] [Q&A]
I tried to run one of the refine examples, so I started by running the following:

`npm create refine-app@latest -- --example app-crm`

Then I faced the following error:

![image](https://github.com/refinedev/refine/assets/85762931/ac24d886-1bc8-4464-920f-d8d0bed2760e)

Runing `npm run start` ended with **sh: 1: refine: not found**. I tried to run `npm i @refinedev/core` but after a minute I got this:

**Killed idealTree:@testing-library/react: sill placeDep ROOT @testing-library/dom@9.3.4 OK for: @testing-library/user-event@14.5.2 want: >=7.21.4**

I'm new to Node.js and NPM, so I don't know what the next step is. Any help would be appreciated.

**Comment by BatuhanW** [comment]
Hello @BabakAmini, couldn't reproduce on my local. Could you check your node version? Ideally, you should use node 18.19.0+.

If you are new to Node.js, I suggest using `nvm`, it's really useful and takes code of node versioning for you. https://github.com/nvm-sh/nvm

![Screenshot 2024-01-30 at 18 08 26](https://github.com/refinedev/refine/assets/16444991/03eff028-af7a-4130-baff-de91bac7f7c2)

**Reply by BabakAmini** [reply]
Hello @BatuhanW and thanks for your reply. My node version is 21.6.1.

**Comment by BabakAmini** [comment]
@BatuhanW I am currently running my code on a remote Ubuntu server, and I attempted to install the mentioned example on Node.js version 20.10.0, but again, it was not successful. However, I successfully installed it on my local Windows 11 machine with the same version of Node. Now when I try to log in, I get this error:

![image](https://github.com/refinedev/refine/assets/85762931/acc6410d-864e-4a86-beb1-00afbcea45e8)
